contracts/HydraChain/modules/ValidatorsData/IValidatorsData.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+struct ValidatorPower {
+    address validator;
+    uint256 votingPower;
+}
+
```

Comment:
Add natspec explanation that the module is a real-time (for the current epoch)  data source about the active validators and their voting power. We need it because the ValidatorsManager shows updates that aren't applied on core consensus level yet.